tests/integration/mixins/track-relationships-test.js

```diff
@@ -215,6 +216,8 @@ module('Integration | Mixin | track relationships as Class', function(hooks) {
   setupRenderingTest(hooks);
 
   hooks.beforeEach(function() {
+    this.owner.register('serializer:application', JSONAPISerializer);
```

Comment:
This is another fix to get the data builds passing.  I missed it in #23 

https://github.com/emberjs/data/runs/4206785231?check_suite_focus=true